packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -0,0 +1,542 @@
+                                  />
+                                ),
+                              }[header.column.getIsSorted() as string] ?? null}
+                            </div>
+                            {header.column.getCanFilter() &&
+                            totalRowCount > UNMANAGEABLE_SUBJECT_COUNT ? (
+                              <div>
+                                <Filter column={header.column} table={table} />
+                              </div>
+                            ) : null}
+                          </>
+                        )}
+                        {header.column.getCanResize() && (
+                          <div
+                            className={`resizer ${
+                              header.column.getIsResizing() ? 'isResizing' : ''
+                            }`}
+                            onMouseDown={header.getResizeHandler()}
+                            onTouchStart={header.getResizeHandler()}
+                          />
+                        )}
+                      </th>
+                    )
+                  })}
+                </tr>
+              ))}
+            </thead>
+            <tbody>
+              {table.getRowModel().rows.map(row => {
+                return (
+                  <tr key={row.id} style={{ height: '30px' }}>
+                    {row.getVisibleCells().map(cell => {
+                      return (
+                        <td
+                          key={cell.id}
+                          style={{
+                            width: cell.column.getSize(),
+                          }}
+                        >
+                          {flexRender(
+                            cell.column.columnDef.cell,
+                            cell.getContext(),
+                          )}
+                        </td>
+                      )
+                    })}
+                  </tr>
+                )
+              })}
+            </tbody>
+          </table>
+        </Box>
+      )}
+      <EntityFinderModal
+        configuration={{
+          selectMultiple: true,
+          selectableTypes: selectableTypes,
+          versionSelection: VersionSelectionType.DISALLOWED,
+          initialScope: FinderScope.ALL_PROJECTS,
+          initialContainer: 'root',
+        }}
+        promptCopy={'Select Entities to add to the Synapse ID list'}
+        show={showEntityFinder}
+        title={'Select Entities'}
+        confirmButtonCopy={'Add Entities'}
+        onConfirm={items => {
+          const newEntityIDsArray = items.map(ref => ref.targetId)
+          const newEntityIDsString =
+            newEntityIDs.trim().length > 0
+              ? newEntityIDs.concat(',')
+              : newEntityIDs
+          const newValue = newEntityIDsString.concat(
+            newEntityIDsArray.join(','),
+          )
+          updateNewEntityIDs(newValue)
+          setShowEntityFinder(false)
+        }}
```

Comment:
It's super weird to me to not just add these after confirming. I think we should make it a long-term goal to have design investigate flipping this and adding a "Paste list of IDs" to the Entity Finder, instead of using the entity finder to maybe populate a text field.